sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/QueryMetrics.java

```diff
@@ -295,6 +296,6 @@ public static QueryMetrics createFromDelimitedStringAndClientSideMetrics(String
 
     @Override
     public String toString() {
-        return toTextString(0);
+        return RntbdObjectMapper.toString(this);
```

Comment:
Same here, if there is no objectMapper, we should use the `Utils.getObjectMapper()`

Reply:
does this also fails on serialization if we try to use the `Utils.getObjectMapper()`?

Reply:
I've changed the objectMapper config in FeedResponse to handle Duration types.